Lib/asyncio/unix_events.py

```diff
@@ -195,32 +195,34 @@ def _make_write_pipe_transport(self, pipe, protocol, waiter=None,
     async def _make_subprocess_transport(self, protocol, args, shell,
                                          stdin, stdout, stderr, bufsize,
                                          extra=None, **kwargs):
-        with events.get_child_watcher() as watcher:
-            if not watcher.is_active():
-                # Check early.
-                # Raising exception before process creation
-                # prevents subprocess execution if the watcher
-                # is not ready to handle it.
-                raise RuntimeError("asyncio.get_child_watcher() is not activated, "
-                                   "subprocess support is not installed.")
-            waiter = self.create_future()
-            transp = _UnixSubprocessTransport(self, protocol, args, shell,
-                                              stdin, stdout, stderr, bufsize,
-                                              waiter=waiter, extra=extra,
-                                              **kwargs)
-
-            watcher.add_child_handler(transp.get_pid(),
-                                      self._child_watcher_callback, transp)
-            try:
-                await waiter
-            except (SystemExit, KeyboardInterrupt):
-                raise
-            except BaseException:
-                transp.close()
-                await transp._wait()
-                raise
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', DeprecationWarning)
```

Comment:
Hm, the extra indent level feels undesirable. Maybe we should have a private function that gets the watcher without triggering the warning? (I'd suggest refactoring get_child_watcher() but that seems tricky given that we must still support a 3rd party policy that provides a custom watcher for several releases.)

Reply:
This would probably emit even more warnings when policy system will be deprecated, so best left to cover entire block. This code will be rewritten anyways (probably by me) when we remove the child watcher system. 

> (I'd suggest refactoring get_child_watcher() but that seems tricky given that we must still support a 3rd party policy that provides a custom watcher for several releases.)

That is indeed tricky since policies override `get_child_watcher` and may not store watcher in `_watcher` so this would break.

Reply:
Okay, let's leave it this way.